data/ka'het/ka'het missions.txt

```diff
@@ -82,7 +82,7 @@ mission "First Contact: Ka'het: Remnant 1B"
 				`	"Wait, why did you call them 'shells' and not ships?"`
 					goto shells
 			label tech
-			`	"Back then we found large numbers of their EMP torpedoes, most of which in almost pristine conditions; their internal mechanisms were also simple enough that we were able to reproduce them perfectly. We didn't find any other suriving systems in the 'shells' to do the same, but their tech have been our main inspiration point for centuries."`
+			`	"Back then we found large numbers of their EMP torpedoes, most of which in almost pristine conditions; their internal mechanisms were also simple enough that we were able to reproduce them perfectly. We didn't find any other suriving systems in the 'shells' to do the same, but their tech has been a source of inspiration for centuries."`
```

Comment:
`most of which in` sounds odd, maybe `most in`?

`suriving` -> `surviving`

Why is `shells` quoted in this line?

Reply:
I think it is because they are actually ships, not shells. But the ships serve the same purpose that a shell would serve for something like a snail.

Reply:
Also, fixed:
suriving -> surviving
most of which in -> most in

Reply:
I bring up the quoting of shells by Taely since it isn't something that she is stressing or quoting - it's normal. When the player responds to the usage of shells, then the emphasis on the word is correct.

Reply:
OK, removed the quotes, as I agree with your comment. Also reworked the sentence a bit.